### DISCUSSION TITLE
Get all flags from unleash

### DISCUSSION THREAD

**Discussion by kimiaShahbaghi** [Unleash/unleash#4188] [Q&A]
I'm using @unleash/proxy-client-react package I know that we can get specific flag from unleash by "useVariant" or "useFlag". is there a way that we can get all flags at the same time from unleash or have access to unleash api response?

**Comment by gastonfournier** [comment]
Hi @kimiaShahbaghi would you mind giving us some context on what you're trying to accomplish here? What's the use case?
